### DISCUSSION TITLE
topic size 0 suddenly

### DISCUSSION THREAD

**Discussion by northboys** [strimzi/strimzi-kafka-operator#6936] [Q&A]
total size is 4151035 but after a few days it suddenly becomes 0, the kafka pod working fine without restart, how to find out why the topic size is 0?.
- Strimzi Operator Version 0.28
<img width="1400" alt="Screen Shot 2022-06-14 at 19 53 33" src="https://user-images.githubusercontent.com/38155944/173581863-432bedd4-2a07-475e-a6a9-e303740b2925.png">

<img width="1203" alt="Screen Shot 2022-06-14 at 13 31 06" src="https://user-images.githubusercontent.com/38155944/173509491-79c104ab-6c09-4296-a2e6-64f28c99862f.png">

**Comment by wengkee** [comment]
Could it be that the messages are older than the retention period set and hence it is deleted?
